public/js/test/integration/todomvc.js

```diff
@@ -50,21 +60,37 @@ function debuggee(callback) {
     return source.slice(firstCurly + 1, -1).trim();
   }
 
-  cy.request("POST", "http://localhost:9002/command", {
-    command: getFunctionBody(callback)
+  return cy.request("POST", "http://localhost:9002/command", {
+    command: getFunctionBody(callback),
+    timeout: timeout
   });
 }
 
 describe("Todo MVC", function() {
+  before(function() {
+    cy.request("POST", "http://localhost:9002/start");
+  });
+
+  after(function() {
+    cy.request("POST", "http://localhost:9002/stop");
+  });
+
   beforeEach(function() {
     debuggee(() => {
-      driver.get("http://todomvc.com/examples/backbone/");
+      driver.get("http://localhost:9002/todomvc/examples/backbone/");
     });
-    cy.visit("http://localhost:8000/#tab=tab1");
+
+    cy.visit("http://localhost:8000/babel");
+
+    cy.get(".tab").first().click();
```

Comment:
clicking the tab handles the case where you refresh the debugger before firefox is open.